service/src/main/resources/db/migration/V1.59__Beacon_Overview_View.sql

```diff
@@ -0,0 +1,14 @@
+CREATE OR REPLACE VIEW beacon_overview AS
+SELECT
+    l.id,
+    l.hex_id,
+    l.last_modified_date
+FROM
```

Comment:
This currently isn't used, but thought it could be helpful having this in the db for any future things we may need. I write it initially to use it, but ran into some potential issues with the restful search api.